maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/internal/EnforcerRuleDesc.java

```diff
@@ -45,7 +45,8 @@ public EnforcerRuleDesc(String name, EnforcerRuleBase rule) {
     }
 
     public String getName() {
-        return name;
+        String ruleName = rule.getRuleName();
+        return ruleName == null || ruleName.isEmpty() ? name : ruleName;
```

Comment:
Names with only whitespace are allowed as `isEmpty()` only checks `length == 0`. `isBlank()` (JDK 11?) implies `trim()` I suggest to disallow whitespace rule names.

Reply:
content is pom tags are trimed by default